src/HypothesisClient/ApiClient/UsersClient.php

```diff
@@ -52,11 +53,54 @@ public function updateUser(
             Uri::fromParts([
                 'path' => 'users/'.$id,
             ]),
-            $headers,
+            $this->getAuthorizationBasic() + $headers,
             json_encode(array_filter([
                 'email' => $email,
                 'display_name' => $display_name,
             ]))
         );
     }
+
+    public function getUserToken(
+        array $headers,
+        string $id
+    ) : PromiseInterface {
+        return $this->postRequest(
+            Uri::fromParts([
+                'path' => 'token',
+            ]),
+            $headers,
+            json_encode([
+                'form_params' => [
+                    'grant_type' => 'urn:ietf:params:oauth:grant-type:jwt-bearer',
+                    'assertion' => $this->getCredentials()->getJWT($id),
+                ],
+            ])
+        );
+    }
+
+    public function getUserAnnotations(
```

Comment:
Could be in a separate client? Doesn't need to be in the same one as user management?

Reply:
These methods should probably match the API end points, so this should be a search method with an optional `user` parameter (and the client itself is configured with a token or not).